qt/python/mantidqt/widgets/plotconfigdialog/curvestabwidget/curves_tab.ui

```diff
@@ -63,20 +63,7 @@
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
-      <widget class="QComboBox" name="select_curve_combo_box">
-       <property name="sizePolicy">
-        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-         <horstretch>0</horstretch>
-         <verstretch>0</verstretch>
-        </sizepolicy>
-       </property>
-       <property name="minimumSize">
-        <size>
-         <width>250</width>
-         <height>0</height>
-        </size>
-       </property>
-      </widget>
+      <widget class="QListWidget" name="select_curve_list"/>
      </item>
      <item>
       <widget class="QPushButton" name="remove_curve_button">
```

Comment:
this button looks squished to me but it's an easy fix. Your maximum size for height is 22 for some reason. Either delete the maximum size property yourself or go into qt creator, select the remove curve button, and for the maximum size property click the little arrow thing which should set it to default values

(Since you haven't changed it I can't comment on the actual code that needs changing but it's line 82)